src/components/UsaBanner/UsaBanner.scss

```diff
@@ -0,0 +1,24 @@
+@use "../../assets/theme/macbis-colors" as c;
+
+// TODO: These colors do not exist in _macbis-colors
+// How and do we incorporate these? Should these be converted to existing colors?
+$black: #1b1b1b;
```

Comment:
Would the `base` color in the Figma (`#212121`) work in place of this color? There also appear to be a few gray options in that palette as well but I don't know how funky they'd look in this context.

Reply:
That's something I want to talk to Michael about, and that's really want my comment on line 4 is meant to address.

Given that this is a net-new design I want to honor what he's got here. If we're going to expand the color palate I want to know how we should be treating these colors before adding them to the macbis-colors stylesheet.

Reply:
Confirmed with Michael this morning that we're going to keep the colors used in the design. This is most likely a one-off and if we have make changes in the future we'll cross that bridge. Updated comment to reflect the usage.